<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SHISSARI-K92MT" failures="2" tests="6" name="newTesting" time="0.056" errors="0" timestamp="5 Oct 2019 13:21:14 GMT">
  <testcase classname="testpackage.Retry" name="testOne" time="0.014">
    <failure type="java.lang.AssertionError" message="display the message expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: display the message expected [true] but found [false]
	at testpackage.Retry.testOne(Retry.java:15)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testOne -->
  <testcase classname="testpackage.Retry" name="testThree" time="0.001"/>
  <testcase classname="testpackage.Retry" name="testTwo" time="0.001">
    <skipped/>
  </testcase> <!-- testTwo -->
  <testcase classname="testpackage.Retry" name="testTwo" time="0.002">
    <skipped/>
  </testcase> <!-- testTwo -->
  <testcase classname="testpackage.Retry" name="testTwo" time="0.007">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at testpackage.Retry.testTwo(Retry.java:32)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testTwo -->
  <testcase classname="testpackage.Retry" name="testing" time="0.002"/>
</testsuite> <!-- newTesting -->
